kinto/plugins/history/listener.py

```diff
@@ -26,7 +27,7 @@ def on_resource_changed(event):
 
     targets = []
     for impacted in event.impacted_records:
-        target = impacted['new']
+        target = impacted['old'] if action == 'delete' else impacted['new']
```

Comment:
I guess that your concern comes from this https://github.com/Kinto/kinto-http.js/blob/3d4069b29b675bbe61d2ce5465abb996f883169a/src/collection.js#L355  

But the right way would be to do ``entry.action == delete`` instead of ``entry.target.data.deleted == true`.

I find the breaking change acceptable though, but open to conversation. Adding `deleted: true` doesn't seem a good idea though :/